client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/InteractionSection/RepetitiveEventForm/RepetitiveEventForm.tsx

```diff
@@ -0,0 +1,108 @@
+
+    const getNextFutureIndex = (prevLastIndex: number) => {
+        return Math.min(lastPossibleIndex,prevLastIndex + EDGE_DAYS_TO_DISPLAY)
+    };
+
+    const getNextPastIndex = (prevFirstIndex: number) => {
+        return Math.max(firstPossibleIndex, prevFirstIndex - EDGE_DAYS_TO_DISPLAY)
+    };
+
+    React.useEffect(() => {
+        if (selectedDates.length === 0) {
+            setSelectedDates([datesToOffer[selectedDateIndex]]);
+            setFirstDayToDisplay(getNextPastIndex(selectedDateIndex));
+            setLastDayToDisplay(getNextFutureIndex(selectedDateIndex))
+        }
+    }, [selectedDateIndex]);
+
+    const loadNextDays = () => {
+        setLastDayToDisplay(getNextFutureIndex)
+    };
+
+    const loadPreviousDays = () => {
+        setFirstDayToDisplay(getNextPastIndex)
+    };
+
+    const onDateCheckClick = (date: Date, isChecked: boolean) => {
+        isChecked
+            ? setSelectedDates(prevDates => prevDates.concat(date))
+            : setSelectedDates(prevDates => prevDates.filter(selectedDay => !isSameDay(selectedDay, date)))
+    };
+
+    const isDateSelected = (date: Date) => selectedDates.some(selectedDay => isSameDay(selectedDay, date));
+
+    const DateCheckbox = ({day}: { day: Date }) => (
+        <Checkbox color='primary'
+                  checked={isDateSelected(day)}
+                  onChange={(event, checked) => onDateCheckClick(day, checked)}/>
+    );
+
+    const DateTitle = ({day}: { day: Date }) => (
+        <FormControlLabel
+            onClick={(event) => event.stopPropagation()}
+            onFocus={(event) => event.stopPropagation()}
+            control={<DateCheckbox day={day}/>}
+            label={<DetailsFieldsTitle date={day}/>}
+        />
+    );
+
+    return (
+        <>
+            {
+                firstDayToDisplay > firstPossibleIndex &&
+                <Button variant='text' startIcon={<ArrowDropUp/>} onClick={loadPreviousDays}>
+                    לימים קודמים נוספים
+                </Button>
+            }
+            {
+                datesToOffer.slice(firstDayToDisplay, lastDayToDisplay + 1).map(day =>
+                    isDateSelected(day)
+                        ? <Accordion elevation={0} defaultExpanded>
+                            <AccordionSummary expandIcon={<ExpandMore/>}
+                                              className={classes.formSectionTitle}>
+                                <DateTitle day={day}/>
+                            </AccordionSummary>
+                            <AccordionDetails classes={{root: classes.formSection}}>
+                                <InteractionDetailsFields/>
+                            </AccordionDetails>
+                        </Accordion>
+                        : <div className='.MuiAccordion-root .MuiAccordionSummary-root'>
+                            <DateTitle day={day}/>
+                        </div>
```

Comment:
move the div to DateTitle

Reply:
it's irrelevant to the second use of DateTitle